CAIPs/caip-294.md

```diff
@@ -127,6 +127,83 @@ window.addEventListener("caip294:wallet_prompt", (event) => {
 });
 ```
 
+### Wallet Data
+
+The `walletData` object MUST include the following properties:
+
+- `uuid`: A unique identifier for the wallet instance.
+- `name`: The name of the wallet.
+- `icon`: An icon representing the wallet.
+- `rdns`: The reverse domain name of the wallet provider.
+
+Additionally, the `walletData` object MAY include the following optional properties:
+
+- `extensionId`: The canonical extension ID of the wallet provider for the active browser.
+- `scopes`: An object defining the authorization scopes supported by the wallet, as specified in CAIP-217.
```

Comment:
here, too, I'm wondering if we want `scopeStrings` or just CAIP-2s... because here it might actually be useful to have a wildcard (as in, `eip155` to mean `any public or private network`, as opposed to a finite series of chainIds implying any others can't be connected).

Reply:
also, why an object and not an array of strings? I think the semantics here are worth pinning down: is this an exhaustive list, or a recommended/representative subset?

Reply:
Discussed on today's call: if these are for discovery and capability expression, the semantics should probably be different and explained in this spec, lest dapps interpret these as pre-declarations.  we have ideas to discuss next week!